R/IDateTime.R

```diff
@@ -240,6 +240,30 @@ rep.ITime = function (x, ...)
   class(y) = "ITime"   # unlass and rep could feasibly not copy, hence use class<- not setattr()
   y
 }
+                           
+round.ITime <- function(x, digits = c("hours", "minutes")) 
+{
+  (setattr(switch(match.arg(digits),
+                  hours = as.integer(round(unclass(x)/3600L)*3600L),
```

Comment:
`3600` here is fine; `round` output will be numeric already, I imagine it will be slightly faster to multiply `double*double` than `double*(double)int` (i.e. casting `int`->`double` at C level)